_config/forms.yml

```diff
@@ -1,6 +1,11 @@
 ---
 Name: adminforms
 ---
+SilverStripe\Admin\LeftAndMain:
+  admin_themes:
+    - 'silverstripe/admin:cms-forms'
+    - '$default'
+
```

Comment:
Please move this block into a new file `themes.yml` and give it `Name: adminthemes`, thanks.

Reply:
:heavy_check_mark: